site-content/source/modules/ROOT/pages/development/release_process.adoc

```diff
@@ -85,13 +160,23 @@ git switch cassandra-<version-branch>
 
 Follow the prompts.
 
+Once artifacts are built and pushed to the staging area, the script will pause and it will ask you to go to
+https://repository.apache.org/#stagingRepositories[Staging repositories] and you will find the repository there. 
+You need to select Cassandra repository and push the "Close" button. 
+Please take a note of a number of that repository like `orgapachecassandra-1283` - number is 1283. 
+This will be asked by the script to proceed. It will use this number upon rendering the e-mail template to send to dev list etc.
```

Comment:
```suggestion
The script will require the repo number to proceed. It will use this number in the rendered e-mail template sent to the dev list, etc.
```